CHANGES/669.misc

```diff
@@ -0,0 +1 @@
+move update type detecting to Update
```

Comment:
Changelog should describe what changes have been made and not what is changing at the moment
```suggestion
Update type detection is was moved from Dispatcher to Update object
```

In this PR you make a movement of the functionality, so in the changelog it should be described as already moved or already added and etc.